website/docs/auth/service_principal_client_secret.html.markdown

```diff
@@ -111,41 +117,98 @@ $ az vm list-sizes --location westus
 $ az account list-locations
 ```
 
+Finally, since we're logged into the Azure CLI as a Service Principal we recommend logging out of the Azure CLI (but you can instead log in using your user account):
+
+```bash
+$ az logout
+```
+
+Information on how to configure the Provider block using the newly created Service Principal credentials can be found below.
+
+---
+
 ### Creating a Service Principal in the Azure Portal
 
-There are two tasks needed to create a Service Principal via [the Azure Portal](https://portal.azure.com):
+There are three tasks needed to create a Service Principal via [the Azure Portal](https://portal.azure.com):
 
  1. Create an Application in Azure Active Directory (which acts as a Service Principal)
- 2. Grant the Application access to manage resources in your Azure Subscription
+ 2. Generating a Client Secret for the Azure Active Directory Application (which can be used for authentication)
+ 3. Grant the Application access to manage resources in your Azure Subscription
 
 ### 1. Creating an Application in Azure Active Directory
 
 Firstly navigate to [the **Azure Active Directory** overview](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/Overview) within the Azure Portal - [then select the **App Registration** blade](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/RegisteredApps/RegisteredApps/Overview) and click **Endpoints** at the top of the **App Registration** blade. A list of URIs will be displayed and you need to locate the URI for **OAUTH 2.0 AUTHORIZATION ENDPOINT** which contains a GUID. This is your Tenant ID / the `tenant_id` field mentioned above.
```

Comment:
Does "This is your Tenant ID (the tenant_id field mentioned above)." seem clearer than "This is your Tenant ID / the tenant_id field mentioned above."?

Reply:
(as above)